packages/base/src/3dview/mainview.tsx

```diff
@@ -377,6 +399,7 @@ export class MainView extends React.Component<IProps, IStates> {
     this._renderer.setRenderTarget(null);
     this._renderer.clearDepth();
     this._renderer.render(this._scene, this._camera);
+    this._viewHelper.render(this._renderer);
```

Comment:
What happens if you add the viewhelper to the scene instead of calling this? (that way it gets renderer with the scene)

Reply:
This way viewHelper gets added to the scene and it's no longer independent of the scene panning, plus by default it gets added to the center of the scene like this - 
![image](https://github.com/user-attachments/assets/dfbdc180-9af2-41d1-a8af-45ebcb725969)